.buildkite-bazelrc

```diff
@@ -37,8 +37,13 @@ build --flaky_test_attempts=5
 # Enable kafka for CI tests only.
 test --config=kafka_enabled
 
+# Enable blst for CI tests also.
+test --config=blst_enabled
+
 build --bes_backend=grpcs://builds.prylabs.net:1985
 build --bes_results_url=https://builds.prylabs.net/invocation/
+build --remote_cache=grpcs://builds.prylabs.net:1985
+build --noremote_upload_local_results
```

Comment:
Is this necessary? It can cause flaky issues.

```suggestion
```

Reply:
Oh no idea about that, I dont think i changed it. I can revert it